synapse/handlers/typing.py

```diff
@@ -86,6 +89,7 @@ def _reset(self) -> None:
         self._member_last_federation_poke = {}
         self.wheel_timer = WheelTimer(bucket_size=5000)
 
+    @wrap_as_background_process("typing.run_as_background_process")
```

Comment:
Should this be `"typing._handle_timeouts"` or similar instead?

Reply:
... yes